### DISCUSSION TITLE
Welcome to flutter_dmzj Discussions!

### DISCUSSION THREAD

**Discussion by xiaoyaocz** [xiaoyaocz/flutter_dmzj#85] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by julewater** [comment]
大佬加油

**Comment by NaiExios** [comment]
大哥！近期出bug了！经常加载不出漫画详情！历史记录也看不了！
